libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -495,8 +483,8 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
   }
 
   get _canNavigateForward(): boolean {
-    const reachedFutureDatesLimit =
-      this.disableFutureDates && isSameMonth(this.activeMonth, this.todayDate);
+    const today = this.todayDate ? startOfDay(this.todayDate) : startOfDay(new Date());
```

Comment:
Consider simplifying:
```suggestion
    const today = startOfDay(this.todayDate ?? new Date());
```